### PR TITLE
[Vision] Add technical soundness and architectural integrity to Thorough Review

### PR DESCRIPTION
**Pull request by fantasai**
Addresses #170 #141

***
<a href="https://pr-preview.s3.amazonaws.com/fantasai/AB-public/pull/173.html" title="Last updated on Sep 27, 2024, 5:20 PM UTC (919d995)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/AB-public/173/dfce969...fantasai:919d995.html" title="Last updated on Sep 27, 2024, 5:20 PM UTC (919d995)">Diff</a>

### CODE REVIEW

**Comment by cwilso** [comment]
I would rather view this as #170 fixes #141, and only mention "technical soundness".

**Comment by fantasai** [comment]
@cwilso I'm not sure I understand your comment, but I think these are two different (certainly related) things:
- technical soundness is something we ensure through the deep review that happens through our WG discussions, and through our implementation experience requirements -- ensuring out specs are thorough, detailed, well-thought out
- architectural integrity is about ensuring the whole system hangs together, and we do it mainly through TAG review and that kind of thing

We do both, and both are valuable, but they are distinct; so that's why I kept them distinct in the PR.

**Review by tantek — APPROVED** [review]
Upon deeper reflection and a bit of research of prior wording of these things at W3C (multiple sources), I have edited the PR to a single line to encapsulate both in a shorter phrase that has established use at W3C.

I do agree with the use-cases / motivations in both issues @fantasai has mentioned and believe this edit now reinforces and resolves both.

**Comment by fantasai** [comment]
@tantek I don't think that encompasses both, because you can have good architecture and sloppy details.

**Comment by fantasai** [comment]
[reverted to the original commit; please don't edit someone else's PR, it's very weird to have one's proposal unilaterally replaced with someone else's proposal]

**Comment by tantek** [comment]
> [reverted to the original commit; please don't edit someone else's PR, it's very weird to have one's proposal unilaterally replaced with someone else's proposal]

Sorry this was me being clumsy with GitHub UI and attempting to suggest edits to a PR and clicking the wrong series of buttons / not understanding what I was doing. :/

File: Vision/Vision.bs [hunk 1]
```diff
@@ -148,6 +148,8 @@ Markup Shorthands: markdown yes
 		accessibility, 
 		internationalization,
 		sustainability,
+		technical soundness,
+		architectural integrity,
 		privacy,
 		and security.
```
Comment (tantek) [inline Vision/Vision.bs:154]:
```suggestion
		security, 
		technical soundness, 
		and architectural integrity.
```

File: Vision/Vision.bs [hunk 2]
```diff
@@ -148,6 +148,8 @@ Markup Shorthands: markdown yes
 		accessibility, 
 		internationalization,
 		sustainability,
+		technical soundness,
```
Comment (tantek) [inline Vision/Vision.bs:151]:
```suggestion
```

File: Vision/Vision.bs [hunk 3]
```diff
@@ -148,6 +148,8 @@ Markup Shorthands: markdown yes
...
+		architectural integrity,
```
Comment (tantek) [inline Vision/Vision.bs:152]:
```suggestion
```

**Review by tantek — CHANGES_REQUESTED** [review]
Suggest placing these additions at end of that list, both per cited examples, and perhaps more importantly following some of the priority of constituencies:
* users (a11y, i18n, s12y, privacy, security) 
* over implementers (technical soundness) 
* over theoretical purity (architectural integrity).

**Comment by fantasai** [comment]
@tantek Done!

**Review by tantek — APPROVED** [review]

**Review by cwilso — APPROVED** [review]

**Pull request merged**